Snapper/Manager.py

```diff
@@ -0,0 +1,74 @@
+from Snapper.Snapper import Snapper
+from Snapper.AssociationMatrix import AssociationMatrix
+from Relationship.Variable import Variable
+from Relationship.SpearframeRelationship import SpearframeRelationship
+
+
+class Manager:
+
+    def __init__(self):
+        self.sensors = []
+        self.route_map = {}
+        self.variables = []
+        self.snapper = Snapper(self)
+        self.matrix = AssociationMatrix()
+
+    def add_sensor(self, sensor):
```

Comment:
The whole point in the manager is to black box as much of the underlying system as possible.  Construction with a known snapper would be repetitive for each sensor when some one was utilizing our engine, and also defeats the point in the black box.

I'm fine with going ahead in this PR making changes Sensor to better fit everything.